index.bs

```diff
@@ -1827,49 +1827,93 @@ To <dfn for="WebTransportError">set up</dfn> a {{WebTransportError}} |error| wit
+not be immediate due to buffering.
 
   <table class="data">
     <colgroup class="header"><col></colgroup>
     <colgroup><col></colgroup>
     <thead>
       <tr>
-        <th>Method</th>
+        <th>API Method</th>
         <th>QUIC Protocol Action</th>
       </tr>
     </thead>
     <tbody>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.{{WritableStream/abort}}(errorCode)</td>
-        <td>send RESET_STREAM with errorCode</td>
+        <td>[=stream/Reset|sends RESET_STREAM=] with errorCode</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.{{WritableStream/close}}()</td>
-        <td>send STREAM_FINAL</td>
+        <td>[=stream/Send|sends=] STREAM_FINAL</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.getWriter().{{WritableStreamDefaultWriter/write}}()</td>
-        <td>send STREAM</td>
+        <td>[=stream/Send|sends=] STREAM</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.getWriter().{{WritableStreamDefaultWriter/close}}()</td>
-        <td>send STREAM_FINAL</td>
+        <td>[=stream/Send|sends=] STREAM_FINAL</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/writable}}.getWriter().{{WritableStreamDefaultWriter/abort}}(errorCode)</td>
-        <td>send RESET_STREAM with errorCode</td>
+        <td>[=stream/Reset|sends RESET_STREAM=] with errorCode</td>
       </tr>
       <tr>
         <td>{{WebTransportBidirectionalStream/readable}}.{{ReadableStream/cancel}}(errorCode)</td>
-        <td>send STOP_SENDING with errorCode</td>
+        <td>[=send STOP_SENDING|sends STOP_SENDING=] with errorCode</td>
       </tr>
       <tr>
-        <td>{{WebTransportBidirectionalStream/readable}}.getReader().{{ReadableStreamDefaultReader/read}}()</td>
-        <td>receive STREAM or STREAM_FINAL</td>
+        <td>{{WebTransportBidirectionalStream/readable}}.getReader().{{ReadableStreamGenericReader/cancel}}(errorCode)</td>
+        <td>[=send STOP_SENDING|sends STOP_SENDING=] with errorCode</td>
       </tr>
       <tr>
-        <td>{{WebTransportBidirectionalStream/readable}}.getReader().{{ReadableStreamGenericReader/cancel}}(errorCode)</td>
-        <td>send STOP_SENDING with errorCode</td>
+        <td>wt.{{WebTransport/close}}(closeInfo)</td>
+        <td>[=session/terminate|terminates=] connection with closeInfo<br>
+      </tr>
+    </tbody>
+  </table>
+
+  <table class="data">
+    <colgroup class="header"><col></colgroup>
+    <colgroup><col></colgroup>
+    <thead>
+      <tr>
+        <th>QUIC Protocol Action</th>
+        <th>API Effect</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>received [=stream-signal/STOP_SENDING=] with errorCode</td>
+        <td>[=WritableStream/Error|errors=] {{WebTransportBidirectionalStream/writable}}
+        with {{WebTransportError/streamErrorCode}}</td>
```

Comment:
Should this be `errorCode`?

Reply:
Whoops forgot this one. Let me push that fix

Reply:
@yutakahirano Actually this reads as I intended. I wanted to show that on [reception of STOP_SENDING](https://pr-preview.s3.amazonaws.com/jan-ivar/webtransport/pull/407.html#send-stream-STOP_SENDING) the "application protocol error code" (a bit lengthy for a table perhasp) maps to [streamErrorCode](https://w3c.github.io/webtransport/#dom-webtransporterror-streamerrorcode). Maybe we can address it in a new issue along with the your existing concern that errorCode in this table isn't exactly the same in the left vs right columns of these two tables?